log4j-core/src/main/java/org/apache/logging/log4j/core/config/json/package-info.java

```diff
@@ -18,7 +18,7 @@
  * Classes and interfaces supporting configuration of Log4j 2 with JSON.
  */
 @Export
-@Version("2.20.1")
+@Version("2.26.0")
```

Comment:
@yybmion, just a sec, why do we need these version bumps? We did not touch to these packages.